apps/webservice/src/app/[workspaceSlug]/_components/CreateRelease.tsx

```
@@ -243,110 +233,72 @@ export const CreateReleaseDialog: React.FC<{
-                      control={form.control}
-                      name={`releaseDependencies.${index}.targetMetadataGroupId`}
-                      render={({ field: { value, onChange } }) => (
-                        <FormItem className="col-span-1">
-                          <Select value={value} onValueChange={onChange}>
-                            <SelectTrigger className="h-8 text-sm">
-                              <SelectValue placeholder="Metadata Group" />
-                            </SelectTrigger>
-                            <SelectContent>
-                              <SelectGroup>
-                                {targetMetadataGroups.data?.map((group) => (
+                <div className="flex items-center gap-2">
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)